.github/workflows/publish.yml

```diff
@@ -52,29 +49,25 @@ jobs:
           echo "##[set-output name=version;]$(echo ${GITHUB_REF#refs/tags/})"
 
       - name: Build apisix RPM Package
-        if: ${{ startsWith(steps.tag_type.outputs.version, 'apisix/') }}
+        if: ${{ startsWith(steps.tag_type.outputs.version, 'apisix/') || startsWith(steps.tag_type.outputs.version, 'build-all/') }}
         env:
-          APISIX_TAG_VERSION: ${{ steps.tag_env.outputs.version}}
+          APISIX_TAG_VERSION: 3.3.0
         run: |
           # build apisix
-          make package type=rpm app=apisix openresty=apisix-base checkout=${APISIX_TAG_VERSION} version=${APISIX_TAG_VERSION} image_base=centos image_tag=7
```

Comment:
Why does this workflow remove building for el7 and el8?

Reply:
it's just POC, for quick build a rpm of redhat